assets/src/features/home-page/components/VideoroomHomePage.tsx

```diff
@@ -59,12 +59,20 @@ const VideoroomHomePage: FC = () => {
 
   const navigate = useNavigate();
   const { audio, video } = useLocalPeer();
+  const wasVideoStarted = useRef(false);
+  const wasAudioStarted = useRef(false);
```

Comment:
Without these refs the user wouldn't be able to stop the camera or microphone, because the stream would change, and execute the effect below.